tests/test_bookstore/tests/test_action_server.py

```diff
@@ -0,0 +1,77 @@
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+from odoo import fields
+from odoo.tests.common import TransactionCase
+from odoo.tests import tagged, Form
+
+
+@tagged('post_install', '-at_install')
+class BookstoreAutomationsTestCase(TransactionCase):
+    @classmethod
+    def setUpClass(cls):
+        super().setUpClass()
+        cls.product = cls.env['product.template'].create({
+            'name': 'Test Book',
+            'list_price': 20.0,
+        })
+
+    def test_update_inventory_description_automation(self):
+        # Empty description case
+        self.assertEqual(self.product.description_pickingin, '[20.0]')
+        self.product.list_price = 24.99
+        self.assertEqual(self.product.description_pickingin, '[24.99]')
+
+        # Description should be overridden
+        self.product.description_pickingin = 'Updated description'
+        self.product.list_price = 30.0
```

Comment:
Normally the server action is done so that if you name it 
```suggestion
        self.product.description_pickingin = '[24.99] Updated description'
        self.product.list_price = 30.0
```
you should end up with '[30.0] Updated description'
Can you add this case?